src/__tests__/resources/timeSeriesList.unit.spec.ts

```diff
@@ -99,4 +101,38 @@ describe('TimeSeriesList class unit test', async () => {
     expect(fetchedDatapoints).toHaveLength(3);
     expect(fetchedDatapoints[0].datapoints[0].timestamp).toBeDefined();
   });
+
+  describe('shallow copy + JSON.stringify()', () => {
+    const items = [
+      {
+        id: 1,
+      },
+      {
+        id: 2,
+      },
+    ];
+
+    test('timeseries.list().autoPagingToArray()', async () => {
```

Comment:
Bad test description. If this test fails what was the cause of the problem by looking at the logs?